src/Sulu/Bundle/MediaBundle/Media/Storage/LocalStorage.php

```diff
@@ -53,13 +48,16 @@ public function __construct($uploadPath, $segments, $logger = null)
     /**
      * {@inheritdoc}
      */
-    public function save($tempPath, $fileName, $version, $storageOption = null)
+    public function save($tempPath, $fileName, $preferredStorageOptions = null)
```

Comment:
Is this something we really want? The sharding has been introduced to distribute the files evenly among all the folders, this way we don't have this advantage if there is a single file with loads of different versions.